doc/rst/language/spec/conversions.rst

```diff
@@ -12,79 +12,188 @@ expression can be a type expression. We refer to these two types the
 implicit (:ref:`Implicit_Conversions`) or
 explicit (:ref:`Explicit_Conversions`).
 
-   *Open issue*.
-
-   Should Chapel allow for user-defined conversions?
-   If so, how would the user define them?
-
 .. _Implicit_Conversions:
```

Comment:
In the opening paragraph above, I think instead of "possibly changing its value" it is better to say "possibly producing a new value".